lighthouse-core/report/html/renderer/report-renderer.js

```diff
@@ -185,6 +179,13 @@ class ReportRenderer {
    * @return {DocumentFragment}
    */
   _renderReport(report) {
+    const i18n = new I18n(report.configSettings.locale, {
+      // Set missing renderer strings to default (english) values.
+      ...Util.UIStrings,
+      ...report.i18n.rendererFormattedStrings,
+    });
+    Util.i18n = i18n;
```

Comment:
I'm a little concerned that somewhere, someday we're going to do the `const strings = Util.i18n.strings` shortcut in a place that doesn't happen on every render and we're going to have a subtle bug we'll probably never catch, but I don't have *great* suggestions around this, `.strings` returns an object that proxies properties to the latest Util.i18n._strings prop? 

any thoughts on this?

Reply:
that would require stashing that value in what is effectively a global (if it isn't being reset every render i assume you're talking about a global). I think good PRs will prevent that.

Reply:
> if it isn't being reset every render i assume it must be a global

This assumption during reviewing is another reason I worry about this :)